js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -721,7 +725,7 @@ const KeyboardNavigationController = core.ViewController.inherit({
     // #endregion Key_Handlers
 
     // #region Click_Handler
-    _pointerDownHandler: function(e) {
+    _clickHandler: function(e) {
```

Comment:
It is not real click. So, I think that will be better to rename it to _focusClickHandler or something like that

Reply:
This handler is not only for focus, if editing enabled e.g.

Reply:
Renamed: _clickHandler -> _pointerEventHandler